app/src/main/java/org/simple/clinic/home/patients/PatientsScreenController.kt

```diff
@@ -72,10 +71,16 @@ class PatientsScreenController @Inject constructor(
         toggleVisibilityOfSyncIndicator(replayedEvents),
         showAppUpdateDialog(replayedEvents),
         showSimpleVideo(replayedEvents),
-        openSimpleVideo(replayedEvents)
+        openSimpleVideo(replayedEvents),
```

Comment:
What happens if there is **both** a celebration, **and** the Simple training video needs to be displayed? Which one has a higher priority?

Reply:
Then could you please add this as a task in another PR to ensure this? This is not taken care of in the current implementation.

Reply:
The current implementation works as expected

Reply:
I believe it only works because of coincidence. If the query to fetch the illustrations takes longer than the query to show the training video, the illustration will be shown instead.

We should make this explicit.